ios/src/TGGeoPoint.h

```diff
@@ -0,0 +1,27 @@
+//
+//  TGGeoPoint.h
+//  tangram
+//
+//  Created by Karim Naaji on 10/27/16.
+//
+//
+
+#ifndef TGGeoPoint_h
+#define TGGeoPoint_h
```

Comment:
I might have used old versions of GCC in the past where this wasn't supported, but it seems that we're good with with the xcode toolchain on that end.

Udpated for `#pragma` syntax.